resources/zib/zib-MultidisciplinaryTeamMeeting-CarePlan.xml

```diff
@@ -0,0 +1,133 @@
+                <map value="NL-CM:15.2.10"/>
+                <comment value="Plan"/>
+            </mapping>
+        </element>
+    <element id="CarePlan.extension">
+      <path value="CarePlan.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="CarePlan.extension:intentTreatment">
+      <path value="CarePlan.extension" />
+      <sliceName value="intentTreatment" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-MultidisciplinaryTeamMeeting.IntentTreatment" />
+      </type>
+    </element>
+        <element id="CarePlan.status">
+            <path value="CarePlan.status"/>
+            <comment value="Indicates whether the plan is currently being acted upon, represents future intentions or is now a historical record.&#xD;&#xA;&#xD;&#xA;For zib MultidisciplinaryTeamMeeting, this will normally be _active_, but the context may specify a more suitable status."/>
+        </element>
+        <element id="CarePlan.intent">
+            <path value="CarePlan.intent"/>
+            <definition value="Indicates the level of authority/intentionality associated with the care plan and where the care plan fits into the workflow chain.&#xD;&#xA;&#xD;&#xA;For zib MultidisciplinaryTeamMeeting, this will normally be _plan_, but the context may specify a more suitable intent."/>
+        </element>
+        <element id="CarePlan.category">
+            <path value="CarePlan.category"/>
+            <slicing>
+                <discriminator>
+                    <type value="pattern"/>
+                    <path value="$this"/>
+                </discriminator>
+                <rules value="open"/>
+            </slicing>
+            <min value="1"/>
+        </element>
+      <element id="CarePlan.category:multidisciplinaryTeamMeetingCode">
+          <path value="CarePlan.category" />
+          <sliceName value="multidisciplinaryTeamMeetingCode" />
+          <min value="1" />
+          <max value="1" />
+          <patternCodeableConcept>
+              <coding>
+                  <system value="http://snomed.info/sct" />
+                  <code value="423134005" />
+              </coding>
+          </patternCodeableConcept>
+      </element>
+    <element id="CarePlan.encounter">
+      <path value="CarePlan.encounter" />
+      <short value="Plan" />
+      <alias value="Beleid" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Encounter" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-MultidisciplinaryTeamMeeting" />
+      </type>
+      <mapping>
+        <identity value="zib-multidisciplinaryteammeeting-v1.0-2020EN" />
+        <map value="NL-CM:15.2.10" />

```

Comment:
Shouldn't this map to NL-CM:15.2.1 instead, as it is a reference to the Encounter (which is the main resource corresponding to the zib)? If so, short/alias/comment also need to be changed accordingly.

Reply:
I've searched for a precedent, but could not find it. Implemented your suggestion